tasks/task875_emotion_classification.json

```diff
@@ -0,0 +1,19417 @@
+{
+    "Contributors": [
+        "Archan Lagoo"
+    ],
+    "Source": [
+        "emotion (\"https://huggingface.co/datasets/emotion\")"
+    ],
+    "Definition": "In this task, you are given a sentence containing a particular emotion. You must classify the sentence into one of the six emotions. The output must be a string representing a single emotion from sadness, joy, love, anger, fear, and surprise.",
+    "Categories": [
+        "Classification"
+    ],
+    "Positive Examples": [
+        {
+            "input": "i am ever feeling nostalgic about the fireplace i will know that it is still on the property",
+            "output": "love",
+            "explanation": "The word \"nostalgic\" could either indicate \"love\" or \"sadness\", but the fact that it is still on the property eliminates \"love\"."
+        },
+        {
+            "input": "i am feeling grouchy",
+            "output": "anger",
+            "explanation": "The word \"grouchy\" clearly indicates the emotion \"anger\"."
+        }
+    ],

```

Comment:
Please include more positive and negative examples (ideally, one from each emotion).